ui/app/AppLayouts/Chat/popups/community/CommunityProfilePopup.qml

```diff
@@ -123,7 +123,7 @@ StatusModal {
             height: !visible ? 0 : implicitHeight
             enabled: root.contentItem.currentItem.contactListSearch !== undefined && root.contentItem.currentItem.contactListSearch.pubKeys.length > 0
             onClicked: {
-                root.contentItem.currentItem.sendInvites(root.contentItem.currentItem.contactListSearch.pubKeys)
+                root.contentItem.currentItem.sendInvites(root.contentItem.currentItem.contactListSearch.pubKeys, "") // NOTE: empty message
```

Comment:
Shouldn't this send the message?

Reply:
IDK, @jrainville i think this part of code does not being used any more